Sources/StreamChatUI/Utils/DateUtils.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+class DateUtils {
```

Comment:
We should make this component subclassable and injectable (via Components) so users can do whatever they want